stubs/jsonschema/jsonschema/_format.pyi

```diff
@@ -2,14 +2,15 @@ from collections.abc import Callable, Iterable
 from typing import Any, TypeVar
 
 _F = TypeVar("_F", bound=Callable[..., Any])
+_RaisesType = type[Exception] | tuple[type[Exception], ...]
```

Comment:
You've hit a known mypy bug in its handling of PEP-604 type aliases. The workaround is to use the old syntax (`Union`) here instead. The rest of our CI won't complain about it in this instance.

Reply:
Ah, easy enough -- I'll have that done in just a second.

Also, thanks to you and everyone on typeshed for being _super_ responsive! It makes it very easy to contribute. ❤️

Reply:
> thanks to you and everyone on typeshed for being super responsive! It makes it very easy to contribute. ❤️

I'm very glad to hear that! Also, don't beat yourself up too much about the regressions — it's nearly inevitable if you're adding annotations to stubs that currently don't have many type hints at all. One way we could work to reduce these in the future, however, would be to add more open-source projects using `jsonschema` to mypy_primer.

Reply:
I'll give that a shot. [`check-jsonschema`](https://github.com/python-jsonschema/check-jsonschema) is where I'm doing my own work.
And there was a PR from the maintainers of https://github.com/matrix-org/synapse , so perhaps I can integrate that as well. I'll have to see how they have mypy setup, of course.